### ISSUE TITLE
Invalid blur clipping

### ISSUE THREAD

**Issue by JoKalliauer** [RazrFalcon/resvg#241]
#### svg

![svg](https://upload.wikimedia.org/wikipedia/commons/0/05/Vector_saturn.svg)

#### png

|ink|rsvg|resvg|
|--|--|--|
|![Vector saturn](https://user-images.githubusercontent.com/15687581/75484392-010a0380-59a9-11ea-8f84-2cf7c0313407.png)|![Vector saturn_r](https://user-images.githubusercontent.com/15687581/75484398-01a29a00-59a9-11ea-9219-135906684973.png)|![Vector saturn_re](https://user-images.githubusercontent.com/15687581/75484400-023b3080-59a9-11ea-88ad-2deacdfccf8a.png)|

#### Expected
![PNG](https://upload.wikimedia.org/wikipedia/commons/8/85/Vector_saturn_%28Correct_render%29.png)

#### Copyright
source https://commons.wikimedia.org/wiki/File:Vector_saturn.svg

author Kelvinsong

license https://creativecommons.org/licenses/by-sa/3.0/deed.de

**Comment by RazrFalcon** [comment]
Looks like everyone is wrong:

![Vector_saturn](https://user-images.githubusercontent.com/725494/75491693-040fee80-59bf-11ea-94b5-c66a47489cbf.png)

Will investigate further to find the source of the issue.

**Comment by matpaul** [comment]
@RazrFalcon Hi! I try use your implementation of fastblur and saw same effect
fix for me was in `box_blur_vert` change initialization of fv, lv 
```rs
let fv: RGBA8 = backbuf[ti];
let lv: RGBA8 = backbuf[ti + width * (height - 1)]; 
```

in `box_blur_horz` change change initialization of fv, lv
```rs 
let fv: RGBA8 = backbuf[ti];
let lv: RGBA8 = backbuf[ti + width - 1];
```

ps. sorry don't have enough time to PR

**Comment by RazrFalcon** [comment]
Thanks, will try it out.

**Comment by yisibl** [comment]
Hi, @RazrFalcon 

I think the Blur filter might be a separate issue that we encountered in real business and got fixed by @matpaul 's code.

Can we fix the Blur filter issue https://github.com/RazrFalcon/resvg/issues/324 first?

Test case:

![blur-test](https://user-images.githubusercontent.com/2784308/164417284-92ecd11b-741e-44ee-9b79-eb171989a380.svg)

**Comment by RazrFalcon** [comment]
`resvg` produces the correct output according to the SVG spec here.

The image border blur was fixes in v0.33

**Comment by JoKalliauer** [comment]
confirm fix

![Vector_saturn_rendersvg](https://github.com/RazrFalcon/resvg/assets/15687581/e2321cd8-ea36-440d-9e3c-1940fb64b8ce)
